pinot-core/src/test/java/org/apache/pinot/core/util/IngestionUtilsTest.java

```diff
@@ -22,17 +22,27 @@
 import com.google.common.collect.Sets;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
```

Comment:
why is this not in the same project as `IngestionConfigUtils` in spi?

Reply:
+1